### PR TITLE
Doc , update image_lib.rst

### PR DESCRIPTION
**Pull request by brutalcrozt**
improve with add default location when creating thumbnail

### CODE REVIEW

**Comment by brutalcrozt** [comment]
Feel free to give me revision and feedback. I don't mid to make more changes for awesome changes and also awesome documentation. :smile:

File: user_guide_src/source/libraries/image_lib.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ called *mypic.jpg* located in the source_image folder, then create a
 thumbnail that is 75 X 50 pixels using the GD2 image_library. Since the
 maintain_ratio option is enabled, the thumb will be as close to the
 target width and height as possible while preserving the original aspect
-ratio. The thumbnail will be called *mypic_thumb.jpg*
+ratio. The thumbnail will be called *mypic_thumb.jpg* and located at same level with *source_image*.
```
Comment (narfbg) [inline user_guide_src/source/libraries/image_lib.rst]: at **the** same level **as**

Also, please keep it in sync with other line lenghts - we generally start a new line after the 76th character.

**Comment by brutalcrozt** [comment]
@narfbg I already move it to next line. Anything else?


>Also, please keep it in sync with other line lenghts - we generally start a new line after the 76th character.

Did i miss something to read?

**Comment by narfbg** [comment]
What you've quoted was in reference with starting a new line, but you did miss the first part of my comment:

> at **the** same level **as**

Bolded words are corrections ...

Also, please rebase your patch on the latest code. I don't know what you did, but your own commit doesn't show now.

**Comment by brutalcrozt** [comment]
@narfbg sorry i new when using git, really mess up by what is shown on cli and here.
i already change the doc as your request. 
is it now relevant with rebase that you request?
if not more specific rebase command will help me, sorry for this. 
last PR what i made also like this, as i know it about line ending :confused:

**Comment by narfbg** [comment]
Look at the diff and you'll see it's obviously not ok ...

**Pull request closed without merging**